packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -457,14 +457,16 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
             const data = this.getOpsMap.get(result.nonce);
             // Due to socket multiplexing, this client may not have asked for any data
             // If so, there it most likely does not need these ops (otherwise it already asked for them)
-            if (data !== undefined) {
+            // Also we may have deleted entry in this.getOpsMap due to too many requests and too slow response.
```

Comment:
Can we instrument the point where we delete the entry in `getOpsMap` to validate this theory?

Reply:
It is already instrumented - see GetOpsTooMany. But I'll add a bit more info (properties) to connect the two in timeline (i.e. add key ids)